index.html

```diff
@@ -0,0 +1,138 @@
+                            <button style="background-color: #303841;" class="btn">Tribute Page</button>
+                    </div>
+                </div>
+
+                <div class="col-lg-4 col-sm-6 mb-4">
+                    <div class="card">
+                        <div style="background-color: #EEEEEE;" class="card-header my-0">
+                            <a class="text-decoration-none" href=""><h3 class=" my-0 text-center text-dark">Dr. Normal Borlaug</h3></a>
+                        </div>
+                        
+                            <img  class="img-fluid m-0" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
+                            <button style="background-color: #303841;" class="btn">Tribute Page</button>
+                    </div>
+                </div>
+
+                <div class="col-lg-4 col-sm-6 mb-4">
+                    <div class="card">
+                        <div style="background-color: #EEEEEE;" class="card-header my-0">
+                            <a class="text-decoration-none" href=""><h3 class=" my-0 text-center text-dark">Dr. Normal Borlaug</h3></a>
+                        </div>
+                        
+                            <img  class="img-fluid m-0" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
+                            <button style="background-color: #303841;" class="btn">Tribute Page</button>
+                    </div>
+                </div>
+
+                <div class="col-lg-4 col-sm-6 mb-4">
+                    <div class="card">
+                        <img style="height: 250px;"  class="img-fluid " src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt="">
+                        <button style="background-color: #303841;" class="btn">Map Data Across the Globe</button>
+                    </div>
+                </div>
+
+                <div class="col-lg-4 col-sm-6 mb-4">
+                    <div class="card">
+                        <img style="height: 250px;"  class="img-fluid " src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt="">
+                        <button style="background-color: #303841;" class="btn">Map Data Across the Globe</button>
+                    </div>
+                </div>
+
+                <div class="col-lg-4 col-sm-6">
+                    <div class="card">
+                        <img style="height: 250px;"  class="img-fluid " src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt="">
+                        <button style="background-color: #303841;" class="btn">Map Data Across the Globe</button>
+                    </div>
+                </div>
+                
+            </div>
+            <center><button style="background-color: #303841;" class=" border-0 px-4 py-3 mt-3">Show all <i class="fa-solid fa-chevron-right"></i></button></center>
```

Comment:
You should kindly consider using external sheet rather than inline styling.

Reply:
- [ ] Please explain why you are suggesting this change to the code's author. Giving reasons for changes helps the author understand the logic and concept better. It also makes it easy for the author to understand what needs to be done. 😄📗 

#### Example of a clear and easy-to-understand comment that gives reason

 > **_Kindly_** use Stylelint linters for this project. Stylelint reports bad codes in your CSS files. It helps to enforce consistent code and prevents you from making errors in your stylesheets. 👍 😎 
 
 > Kindly put the nav tag into the header tag. The nav tag is a semantic tag that should contain the logo and other navigational links to parts of the site or external sites. 
 
 #### Hint: You don't have to create another inline comment. You could edit the inline comments that you have above. 💻